airdrop-claim-reject-cancel-hts-token-js-sdk/airdrop.js

```diff
@@ -0,0 +1,158 @@
+console.clear();
+import dotenv from "dotenv";
+dotenv.config();
+
+import {
+	Client,
+	PrivateKey,
+	AccountId,
+	Hbar,
+	TokenType,
+	TokenSupplyType,
+	TokenAirdropTransaction,
+	TokenClaimAirdropTransaction,
+	TokenCancelAirdropTransaction,
+	TokenRejectTransaction,
+	AccountBalanceQuery,
+} from "@hashgraph/sdk";
+
+import accountCreateFcn from "./utils/accountCreate.js";
+import * as htsTokens from "./utils/tokenOperations.js";
+import * as queries from "./utils/queries.js";
+
+async function main() {
+	// CONFIGURE ACCOUNTS AND CLIENT, AND GENERATE NEEDED KEYS

```

Comment:
nit: Prefer comments with standard English capitalisation over all-caps.

(applies to other instances throughout the script too, but not going to comment on each one.)

Reply:
N/A